crates/nu-std/std/mod.nu

```diff
@@ -286,3 +286,8 @@ It's been this long since (ansi green)Nushell(ansi reset)'s first commit:
 Startup Time: ($nu.startup-time)
 "
 }
+
+
+export def pwd [] {
```

Comment:
the doc should go just above that line :wink:

Reply:
Done, I planned to add one but forgot 🙈

Let me know if you want more or something different?

Reply:
> Done, I planned to add one but forgot see_no_evil
> 
no worries :wink: 

> Let me know if you want more or something different?
>
no i think it's great like this, it's such a simple command :relieved: 
it's just so that users have at least a short description for all of their commands :ok_hand: